aio/Makefile

```diff
@@ -1,5 +1,5 @@
 all:
-	ocamlbuild -package unix echo.native
+	@dune exec ./echo.bc
```

Comment:
Is this executing the command? I don't think you should execute the command. `echo` server blocks and will not return.